src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_fankriss.cpp

```diff
@@ -87,10 +84,9 @@ struct boss_fankriss : public BossAI
 
     void JustEngagedWith(Unit* who) override
     {
-        _scheduler.CancelAll();
         BossAI::JustEngagedWith(who);
 
-        _scheduler
+        scheduler
             .Schedule(7s, 14s, [this](TaskContext context)
```

Comment:
```suggestion
        scheduler.Schedule(7s, 14s, [this](TaskContext context)
```
Verry misc but it looked weird lol